scripts/benchmark-rule.js

```diff
@@ -6,13 +6,19 @@ const chalk = require("chalk");
 const normalizeRuleSettings = require("../lib/normalizeRuleSettings");
 const postcss = require("postcss");
 const request = require("request");
+const requireRule = require("../lib/requireRule");
 const rules = require("../lib/rules");
 
 const ruleName = process.argv[2];
 const ruleOptions = process.argv[3];
+const ruleContext = process.argv[4];
 
-if (!ruleName || !ruleOptions) {
-  throw new Error("You must specify a rule name and rule options");
+if (!rules.includes(ruleName)) {
+  throw new Error("You must specify a valid rule name");
+}
```

Comment:
I think very nice for splitted `ruleName` and `ruleOptions` checks!

Further, how about replace `rules.includes(ruleName)` like this? 👇 

```js
const ruleFunc = requireRule(ruleName)
if (!ruleFunc) {
  throw new Error("You must specify a valid rule name");
}
```

Because `requireRule()` do `rules.includes(ruleName)` within it; it's doing the same thing.
In addition, by doing so `const rules = require("../lib/rules")` can be removed!

https://github.com/stylelint/stylelint/blob/c8354d8e4cc42b095393b854d3eff9f8987c65df/lib/requireRule.js#L6-L12

Reply:
You are correct, good spot. Not sure how I missed that one. 

Just pushed that change now :+1:

Reply:
Thank you!